### ISSUE TITLE
[CommonVariables] docs don't compile (Trac #1489)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1481]
When I try to compile the docs I don't get past CommonVariables with this (not very helpful) Sphinx error: 

```text
Sphinx error:
'ascii' codec can't decode byte 0xc3 in position 264: ordinal not in range(128)
```

When I delete CommonVariables docs, compilation goes through. This happens both on my local machine (Ubuntu 14.04 and system packages) and on cobalt. 

Migrated from https://code.icecube.wisc.edu/ticket/1489
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "When I try to compile the docs I don't get past CommonVariables with this (not very helpful) Sphinx error: \n\n{{{\nSphinx error:\n'ascii' codec can't decode byte 0xc3 in position 264: ordinal not in range(128)\n}}}\n\nWhen I delete CommonVariables docs, compilation goes through. This happens both on my local machine (Ubuntu 14.04 and system packages) and on cobalt. ", 
    "reporter": "seuler", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[CommonVariables] docs don't compile", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-12-14T16:50:56", 
    "milestone": "", 
    "owner": "mwolf", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151214T16:53:45: 
* seuler changed milestone from "" to "IceRec V5"
* seuler changed summary from "CommonVariables docs don't compile" to "[CommonVariables] docs don't compile"

**Comment by icecube-trac** [comment]
Trac update at 20151214T18:11:11: nega commented:

> do the docs render as expected at: http://software.icecube.wisc.edu/icerec_trunk/projects/CommonVariables/ ?

**Comment by icecube-trac** [comment]
Trac update at 20151214T18:25:57: seuler commented:

> at least they render at all...

**Comment by icecube-trac** [comment]
Trac update at 20151214T18:42:32: jvansanten commented:

> My guess is that the SVN keywords are being expanded in Swedish, and Python is choking on the encoding. Since Python insists on interpreting comments, and SVN uses whatever localization is specified in the environment, there seems to be no robust way to handle this. Options:
> 1. Tell everyone to checkout with `LC_TIME=C` in the environment (this is what the docutils devs do, it turns out)
> 2. Remove SVN keywords from all Python files

**Comment by icecube-trac** [comment]
Trac update at 20160111T16:37:25: 
* nega changed status from "new" to "closed"
* nega changed resolution from "" to "fixed"
* nega commented:

> In changeset 140843:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140843"
> remove keyword expansion from svn properties
> 
> closes #1481
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
